src/swap/SwapScreen.tsx

```diff
@@ -46,15 +46,23 @@ export function SwapScreen() {
   const { t } = useTranslation()
   const dispatch = useDispatch()
 
-  const coreTokens = useSelector(coreTokensSelector)
+  const supportedTokens = useSelector(coreTokensSelector)
+  const supportedTokensAddresses = supportedTokens.map((token) => token.address)
   const swapInfo = useSelector(swapInfoSelector)
+  const tokensSortedByUsdBalance = useSelector(tokensByUsdBalanceSelector)
 
-  const [toToken, setToToken] = useState(
-    coreTokens.find((token) => token.symbol === DEFAULT_TO_TOKEN)
-  )
-  const [fromToken, setFromToken] = useState(
-    coreTokens.find((token) => token.symbol === DEFAULT_FROM_TOKEN)
+  const CELO = supportedTokens.find(
+    (token) => token.symbol === DEFAULT_FROM_TOKEN && token.isCoreToken
   )
+
+  const defaultFromToken =
+    tokensSortedByUsdBalance.find(
+      (token) => token.balance.gt(0) && supportedTokensAddresses.includes(token.address)
```

Comment:
We might higher default for swappable assets something like percent of asset price, but that might be outside of the scope of this PR.

Reply:
The `tokensSortedByUsdBalance` already returns a sorted list of token by usd balance. So this will pick up the supported token with the highest usd balance.
So the `token.balance.gt(0)` is just to avoid the case where the user doesn't have any balance, so we should show CELO as the default token.
Now thinking about it, I think this could introduce a bug, only when the user has supported tokens but we are not able to fetch the price for them. So the sorting won't work correctly, and in that case we will show any of supported tokens.
I will add a `token.balance.gt(0) && token.usdPrice && supportedTokensAddresses.includes(token.address)` to address that case.